### PR TITLE
feat/anvil-manifest

### PR DESCRIPTION
**Pull request by m0nhawk**

### CODE REVIEW

File: qa-anvil.planx-pla.net/portal/gitops.json [hunk 1]
```diff
@@ -115,7 +115,7 @@
   "featureFlags": {
     "explorer": true
   },
-  "arrangerConfig": {
+  "dataExplorerConfig": {
```
Comment (abgeorge7) [inline qa-anvil.planx-pla.net/portal/gitops.json]: remove "node_id" from charts

File: qa-anvil.planx-pla.net/portal/gitops.json [hunk 2]
```diff
@@ -151,30 +151,33 @@
...
-          "fileName": "manifest.json",
-          "dropdownId": "download"
-        }
-      ]
+      "enabled": true
+    },
+    "arrangerConfig": {
+      "projectId": "search",
+      "graphqlField": "case",
+      "index": ""
```
Comment (abgeorge7) [inline qa-anvil.planx-pla.net/portal/gitops.json]: add "nodeCountField": "node_id" to the "arrangerConfig" blob

**Comment by qingyashu** [comment]
I think to use `dataExplorerConfig`, we need to bump portal version in `manifest.json`?

File: qa-anvil.planx-pla.net/portal/gitops.json (hunk 2 above)
Comment (m0nhawk) [reply qa-anvil.planx-pla.net/portal/gitops.json]: I have made the necessary changes.

**Comment by m0nhawk** [comment]
Removed. 👍

**Pull request merged**